package.json

```diff
@@ -128,7 +128,7 @@
     "@data-driven-forms/react-form-renderer": "^3.20.2",
     "@formatjs/cli": "4.8.4",
     "@openshift/dynamic-plugin-sdk": "^3.0.0",
-    "@patternfly/patternfly": "^5.0.0",
+    "@patternfly/patternfly": "^5.0.2",
```

Comment:
The update of PF version is not part of this PR, let's not add complexity to it.